user/user-tool-prefs/tool/src/java/org/sakaiproject/user/tool/UserPrefsTool.java

```diff
@@ -370,14 +370,23 @@ public void setPrefHiddenString(String inp)
 	/**
 	 * @return Returns the prefTimeZones.
 	 */
-	public List getPrefTimeZones()
+	public List<SelectItem> getPrefTimeZones()
 	{
 		if (prefTimeZones.size() == 0)
 		{
 			String[] timeZoneArray = TimeZone.getAvailableIDs();
 			Arrays.sort(timeZoneArray);
-			for (int i = 0; i < timeZoneArray.length; i++)
-				prefTimeZones.add(new SelectItem(timeZoneArray[i], timeZoneArray[i]));
+			for (int i = 0; i < timeZoneArray.length; i++) {
```

Comment:
I'm fine with this but you can replace it will the newer collections streaming. The collection is small so we don't likely get much parallelisation from it, but some think it's easier to read:

			String[] timeZoneArray = TimeZone.getAvailableIDs();
			List<SelectItem> collect = Arrays.stream(timeZoneArray)
					.filter(t -> StringUtils.contains(t, '/'))
					.filter(t -> !(StringUtils.startsWith(t, "SystemV/")))
					.sorted()
					.map(t -> new SelectItem(t, t.replace('_', ' ')))
					.collect(Collectors.toList());
			prefTimeZones.addAll(collect);

Personally I'm on the fence.